### DISCUSSION TITLE
Address bar raises other items on the taskbar when active

### DISCUSSION THREAD

**Discussion by mouless** [valinet/ExplorerPatcher#600] [General]
When I'm not using the Address bar it looks normal:
![image](https://user-images.githubusercontent.com/29571476/147391926-d9bc5980-c72e-4baa-9203-27d10a223529.png)

But when the Address bar is shown it pushes the icons up a bit, making some weird padding under the icons so you can't click on them when the mouse pointer is all the way down on the taskbar (at the bottom of the screen), you'll have to go up a little bit in order to hit them. (The Windows Start-button goes all the way down and does not have this problem).

![image](https://user-images.githubusercontent.com/29571476/147391954-25dbf90c-037d-4ad2-bf23-9dbc1d46adc7.png)

**Comment by valinet** [comment]
This is not an ExplorerPatcher bug. It's an issue with Windows in general. Toolbars were deprecated for ages. The same thing happens in Windows 10 when using a toolbar with large icons and no labels: there is a gap between the icons and the bottom of the bar. The height is large enough to accommodate the tallest element, and the rest just take whatever space they need from top to bottom.

Maybe one day I will take a look an come up with  fix, but it's pretty difficult since testing it is a pain: you have to account for different DPI, different resolutions etc. It would have been better if Microsoft did a good job themselves from the get go.

Also, as I repeat everyday, please post stuff in discussions as per #56.

**Reply by mouless** [reply]
Thank you for the prompt reply and sorry for falsely flagging this as an issue.

I thought it was a bug since I have three other computers running on Win10 with the address bar just like that and neither of them exhibit that behaviour, so it's just puzzling to me.

**Reply by valinet** [reply]
Yeah, no worries, some things are broken in Windows 11. The old interface is supported as much as it is supported. If it’s a bug, it’s not really an EP bug, it’s just the state the interface was left into.
